azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/configurations/RuntimeConfiguration.java

```diff
@@ -20,6 +27,10 @@ public void setOs(String os) {
         this.os = os;
     }
 
+    public OperatingSystemEnum getOperationSystemEnum() throws MojoExecutionException {
```

Comment:
move this mothod to the place where to use  it, change MojoExecutionException at OperatingSystemEnum.fromString to  IllegalArgumentException

Reply:
Moved to `Utils` in maven-plugin-lib